### PR TITLE
Added example for the highlight method.

### PR DESCRIPTION
**Pull request by rahulyesantharao**
Highlights the current tab and the next tab.

### CODE REVIEW

File: tabs-tabs-tabs/tabs.js [hunk 1]
```diff
@@ -122,5 +122,12 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-highlight") { // highlights current tab and next tab
```
Comment (andymckay) [inline tabs-tabs-tabs/tabs.js]: This doesn't actually work on anything but Chrome, so I think we should have some note about that somewhere, maybe in the readme or in the panel itself. 

https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/highlight

File: tabs-tabs-tabs/tabs.js [hunk 2]
```diff
@@ -122,5 +122,12 @@ document.addEventListener("click", function(e) {
...
+    callOnActiveTab((tab, tabs) => {
+      next = (tab.index+1) % tabs.length;
```
Comment (andymckay) [inline tabs-tabs-tabs/tabs.js]: This isn't the next tab. If tab.index + 1 is equal to tabs.length, its 0, which is likely the first tab. I can see the need to pick another tab though, but the comment on the line above is inaccurate and could maybe be improved?

File: tabs-tabs-tabs/tabs.js (hunk 1 above)
Comment (rahulyesantharao) [inline tabs-tabs-tabs/tabs.js]: Oh yeah I noticed that, but wasn't sure what to do about it; I'll add the comment in!

**Comment by rahulyesantharao** [comment]
Hi! I think I made the changes, but I'm not sure if I did it right because I'm new to GitHub.  Please let me know if anything didn't work.  Thanks!

**Comment by andymckay** [comment]
Looks good to me, @wbamberg ?

**Comment by wbamberg** [comment]
Looks great, thanks @rahulyesantharao !

**Pull request merged**